userRoutes.js

```diff
@@ -0,0 +1,53 @@
+const router = require('express').Router();
+const User = require('./userModel');
+
+//GET USERS
+router.get('/users', async (req,res)=> {
+    const user = await User.find();
+    res.send(user)
+})
+
+//GET USERS BY ID
+router.get('/:id', async (req,res)=> {
+    const user = await User.findById();
```

Comment:
User.findById() takes no parameter, there's no way for the DB to retrieve the user for that specific id.